tests/unit/commands/test_remote.py

```diff
@@ -523,6 +526,101 @@ def test_run_in_repo_newer_than_core22(
     emitter.assert_debug("Running new remote-build because base is newer than core22")
 
 
+@pytest.mark.parametrize(
```

Comment:
The content of these tests looks fine, but they look more like integration tests than unit tests to me.

Reply:
True, since it is most test the external library. But I feel put it in spread is too heavy.